maas/client/viscera/nodes.py

```diff
@@ -130,3 +137,14 @@ def as_region_controller(self):
                 'Cannot convert to `RegionController`, node_type is not a '
                 'region controller.')
         return self._origin.RegionController(self._data)
+
+    async def get_power_parameters(self):
+        """Get the power paramters for this node."""
+        data = await self._handler.power_parameters(system_id=self.system_id)
+        return data
+
+    async def set_power_parameters(self, power_type, power_parameters={}):
+        """Set the power parameters for this node."""
+        await self._handler.update(
```

Comment:
You need to use the result from this and update the power_type on this object now, because it has now changed. This should be reflected.